Lib9c/Action/RewardGold.cs

```diff
@@ -23,14 +24,28 @@ public override void LoadPlainValue(IValue plainValue)
 
         public override IAccountStateDelta Execute(IActionContext context)
         {
+            Log.Debug("RewardGold Execute Beginning.");
             var states = context.PreviousStates;
-            states = GenesisGoldDistribution(context, states);
-            states = WeeklyArenaRankingBoard(context, states);
-            return MinerReward(context, states);
+            try
+            {
+                states = GenesisGoldDistribution(context, states);
+                states = WeeklyArenaRankingBoard(context, states);
+                states = MinerReward(context, states);
+            }
+            catch (Exception e)
+            {
+                Log.Debug($"RewardGold throw exception.\n{e.Message}");
```

Comment:
이렇게 하면 traceback도 남아서 더 좋을 것 같습니다.

```suggestion
                Log.Debug(e, $"RewardGold throw exception.\n{e.Message}");
```